### PR TITLE
Create elf_kaiten.txt

### PR DESCRIPTION
**Pull request by MikhailKasimov**
ELF Kaiten/Tsunami static trails to be added to Maltrail upstream. This trail can also be useful to detect Kaiten's derivatives like Linux.Remaiten, Linux.Gafgyt, etc.

Some of addresses are also detected as malicious ASN (OVH), but current trail list makes detection more specific. And it seems good to have some trail list instead of nothing, IMO.

This trail list should be updated manually, because there is no public Kaiten IP list, instead of Linux.Mirai malware ([0] http://sanyalnet-cloud-vps.freeddns.org/mirai-ips.txt , see PR #177). Anyway, I can't find such public list.

Please, consider! Thanks!

### CODE REVIEW

**Comment by MikhailKasimov** [comment]
UPD. BTW, here is an idea for ```elf kaiten (heuristic)``` detection in MalTrail.

[0] Found Kaiten source code on Git: hxxps://github.com/isdrupter/ziggystartux/

In particular main module [1] hxxps://github.com/isdrupter/ziggystartux/blob/master/infector.py , which contains code for IP-addresses to scan and use:

``
class sshscanner(threading.Thread):
    global passwords
    global ipclassinfo
    if ipclassinfo == "A":
        global ip1
    elif ipclassinfo == "B":
        global ip1
        global ip2
    elif ipclassinfo == "C":
        global ip1
        global ip2
        global ip3
    def run(self):
        while 1:
            try:
                while 1:
                    thisipisbad='no'
                    if ipclassinfo == "A":
                        self.host = 'http://'+ip1+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))
                    elif ipclassinfo == "B":
                        self.host = 'http://'+ip1+'.'+ip2+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))
                    elif ipclassinfo == "C":
                        self.host = 'http://'+ip1+'.'+ip2+'.'+ip3+'.'+str(random.randrange(0,256))
                    elif ipclassinfo == "LUCKY":
                        lucky = ["186.115","31.176","113.53","186.113","190.254","190.255","186.114","95.9","95.6","118.174","190.65","203.249","190.66","190.67","122.176","187.109","60.51","186.119","95.169","190.69","190.253","122.168","201.75","117.156","188.59","177.11","182.74","190.68","118.173","190.252","165.229","84.122"]
                        self.host = 'http://'+random.choice(lucky)+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))
                    else:
self.host = 'http://'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))+'.'+str(random.randrange(0,256))
``

And this ```LUCKY``` data seems to be OK to build some heuristic on ```elf kaiten (heuristic)``` algo. What do you think about this? Thanks!

**Comment by stamparm** [comment]
Usage of IPs is as IOCs for malware is highly undesirable as IPs tend to change the owner and the usage. However, domain names are highly desirable. Building of heuristics for (low occurrence) malware is also not desirable.

TLDR; DNS (domain names) are desirable trails in malware cases

**Comment by stamparm** [comment]
I don't see any new stuff related in the last 365 days:



Hence, I am concluding that putting IPs as IOCs or making of heuristics just for it doesn't make much sense

**Comment by MikhailKasimov** [comment]
News in mass-media != reality, especially when someone monitors links to bring undetected samples to AV labs. Yes, Kaiten << Mirai or, even, Gafgyt, but samples are still met rather often.

Especially in case of Kaiten being "father" of derivative crap like Mirai, etc... Some heur seems to be sane. But OK.
